app/src/components/UserAddress.vue

```diff
@@ -2,15 +2,25 @@
   <b-tag rounded class="user-address">
     <span class="address" :title="address">
       <img :src="avatarUrl" class="img mr-1" />
-      <template v-if="truncate">{{ address | truncate(10) }}</template>
-      <template v-else>{{ address }}</template>
+      <a target="_blank" :href="ethersConnect.getNetExplorerUrl() + 'address/' + address">
+        <template v-if="truncate">{{ address | truncate(10) }}</template>
+        <template v-else>{{ address }}</template>
+      </a>
+      <span @click="copyAddress"><b-icon icon="clipboard" type="is-primary" /></span>
     </span>
   </b-tag>
 </template>
 
 <script>
+import ethersConnect from '@/store/modules/ethers/ethersConnect';
+
 export default {
   name: 'UserAddress',
+  data() {
+    return {
+      ethersConnect: ethersConnect,
```

Comment:
Objects inside data option are made reactive by Vue, which is not necessary. You can have a computed property for the explorer URL instead.